content/manuals/build/ci/github-actions/multi-platform.md

```diff
@@ -175,7 +175,7 @@ jobs:
         with:
           platforms: ${{ matrix.platform }}
           labels: ${{ steps.meta.outputs.labels }}
-          outputs: type=image,"name=${{ env.DOCKERHUB_REPO }},${{ env.GHCR_REPO }}",push-by-digest=true,name-canonical=true,push=true
+          outputs: type=image,"name=${{ env.DOCKERHUB_REPO }}:latest,${{ env.GHCR_REPO }}:latest",push-by-digest=true,name-canonical=true,push=true
```

Comment:
No we should not set a tag here, only repo name as we are pushing by digest with `push-by-digest=true` and merging these to create tags in `merge` job. If that fails that means a registry doesn't support push by digest which looks strange to me. I will ask to user habing this issue.

Reply:
I indeed tried adding tags at this step but this leads to strange results as then randomly one of the platform images will have e.g. the latest tag or a semver tag like 0.3.9, which then makes it impossible to use those tags on the other architectures.

Reply:
Added the link to the workflow and the full workflow file to the issue.